pages/index.js

```diff
@@ -1,11 +1,15 @@
 import Layout from '../components/Layout'
 import ScorePostWidget from '../components/ScorePostWidget'
 import ScoreCard from '../components/ScoreCard'
+import Users from '../components/Users'
 import useScores from '../lib/useScores'
 
 const Home = () => {
   const { scores, error } = useScores()
 
+  const usersIds = scores &&
+    [ ...new Map(scores.map(score => [ score.user_id, score.user_name ]).values()) ]
+
```

Comment:
As far as I understand, the end result is a list of pairs [user_id, user_name], which is already accomplished by `scores.map(score => [ score.user_id, score.user_name ])`. Why do we need to call `.values()`, pass it to a new Map, and spread it into an array?

Reply:
The new Map object holds key-value pairs and the key in the Map occurs only once (so we are avoiding duplicates). The `.values()` is not needed, it was left over by mistake.

Reply:
Oh, nice 👍